internal/venice-consumer/src/test/java/com/linkedin/venice/client/consumer/VeniceChangelogConsumerClientFactoryTest.java

```diff
@@ -86,10 +132,10 @@ public void testGetChangelogConsumerThrowsException() {
     ChangelogClientConfig globalChangelogClientConfig =
         new ChangelogClientConfig().setConsumerProperties(consumerProperties).setSchemaReader(mockSchemaReader);
     VeniceChangelogConsumerClientFactory veniceChangelogConsumerClientFactory =
-        new VeniceChangelogConsumerClientFactory(globalChangelogClientConfig);
+        new VeniceChangelogConsumerClientFactory(globalChangelogClientConfig, new MetricsRepository());
     D2ControllerClient mockControllerClient = Mockito.mock(D2ControllerClient.class);
 
-    veniceChangelogConsumerClientFactory.setD2ControllerClient(mockControllerClient);
+    // veniceChangelogConsumerClientFactory.setD2ControllerClient(mockControllerClient);
```

Comment:
If we do not need this mock, then we do not need to mock `mockControllerClient` and no need to prepare `mockControllerClient.getStore(STORE_NAME) ` call below.

Reply:
Good point.